leveldb-server/src/test/db.match.spec.ts

```diff
@@ -0,0 +1,175 @@
+import test, { Macro } from 'ava';
+import { testing } from '../db';
+import { Filter } from '@binaris/shift-interfaces-koa-server/interfaces';
+
+const { match } = testing;
+
+type Operator = 'eq' | 'ne' | 'gt' | 'lt' | 'gte' | 'lte';
```

Comment:
maybe generated this directly in `interfaces/src/db.ts`?

Reply:
Not done.
1. I'd need to break up `Filter` into a union of multiple sub-types, and then take `SubFilter['operator']`.  That break-up would have to go according to what is convenient for this test, not necessarily making sense elsewhere.
2. If a new operator is added, I _anyway_ have to write tests for it.  So I can expand `Operator` at that time.  This is _not_ a type that can just expand because it passes through.